script.js

```diff
@@ -0,0 +1,200 @@
+let webBrands = [];
+let brands=[];
+let resultbrands=[];
+let productNode;
+const proxyurl = "https://cors-anywhere.herokuapp.com/";
+const url = `https://wellspent-ethical.herokuapp.com/api/brands`;
+
+
+var linksList = document.querySelectorAll("[id^=product-]");
+linksList.forEach(function(product) {
+  const node = product.childNodes[0];
+  const label = node.getAttribute("aria-label");
+  let brand = label.toLowerCase();
+  webBrands.push(brand);
+});
+
+
+ // site that doesn’t send Access-Control-*
+fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
+  .then(resultbrandsonse => resultbrandsonse.text())
+  .then(contents => {
+    resultbrands = JSON.parse(contents);
+    brands = resultbrands;
```

Comment:
- There's no point of getting the response.text() and parse it later to JSON, get the JSON format from the first response by replacing **resultbrandsonse.text()** with **resultbrandsonse.json()**.
- On line 22 and 23 I could not get what is the point of having 2 variables with the same content "cloning".
- Variables names do not really describe what they are for, for example, I would use 
   - **response** instead of **resultbrandsonse**
   - **result/data** instead of **contents**  
    and so on as most of the variables names in the code making it hard to understand.